### ISSUE TITLE
Open file/folder not working.

### ISSUE THREAD

**Issue by 108anup** [VSpaceCode/vscode-which-key#26]
## Bug description
Open file/folder command not working

## To Reproduce
1. Install and configure which key extension
2. Press `<spc> f f`
Error: ```command workbench.action.files.openFileFolder not found```

## Expected behavior
Should show a panel for opening files/folders.

## Screenshots
![err](https://user-images.githubusercontent.com/14804163/102111913-f8ce6800-3e04-11eb-9322-92deb0f7b6d8.png)

## Comments
Ideally, I think vscode should be providing this command. Maybe it has been removed from new versions of vscode.
I am on Linux.

**Comment by stevenguh** [comment]
This is because there isn't a cross environment built-in command to open file. In VSpaceCode, we bundled a third party file opener to solve this. In which-key, we are not going to have default bindings that depend on any third party extensions, so I fixed it by adding an extra command in which-key to get around this issue by doing try-catch. (For the record, we probably shouldn't add command not related to the core functionality to which-key. In this case, this is to get around the built-in issue. I see it as a little bit of an exception)
